digest/digest_test.go

```diff
@@ -14,21 +12,6 @@ func TestParseDigest(t *testing.T) {
 		hex       string
 	}{
 		{
-			input:     "tarsum+sha256:e58fcf7418d4390dec8e8fb69d88c06ec07039d651fedd3aa72af9972e7d046b",
-			algorithm: "tarsum+sha256",
```

Comment:
I'd still like to include support for "+" modifier syntax. In particular, it would be nice to support "sha512+trunc384" or "sha256+chunk879". I don't think this PR does it, but we remove all the unit tests that mark support for this.